alca-app/src/components/PositionsTabs.jsx

```diff
@@ -0,0 +1,508 @@
+        },
+    ];
+
+    const lockedPositionsRows = hasLockedPosition
+        ? [
+              {
+                  amount: lockedPosition?.lockedAlca,
+                  id: lockedPosition?.tokenId || 1,
+                  timeLeft: `${
+                      lockedPosition.blockUntilUnlock > 0
+                          ? Number(lockedPosition?.blockUntilUnlock).toLocaleString()
+                          : 0
+                  } Blocks`,
+                  ...lockedPosition,
+              },
+          ]
+        : [];
+
+    const actionButtonStyles = {
+        textTransform: "none",
+        marginRight: 2,
+        fontSize: 13,
+    };
+
+    const defaultTabClasses = {
+        borderRadius: 1,
+        textTransform: "none",
+        fontSize: 14,
+        height: 42,
+        minHeight: "inherit",
+    };
+
+    const currentClasses = {
+        background: `linear-gradient(
+            180deg,
+            ${theme.palette.custom.startGradient} 18.53%,
+            ${theme.palette.custom.endGradient} 167.76%
+        )`,
+        color: theme.palette.background.default,
+    };
+
+    let stakingTabClasses = {
+        ...defaultTabClasses,
+    };
+
+    let positionTabClasses = {
+        ...defaultTabClasses,
+    };
+
+    if (currentTab === "1") {
+        stakingTabClasses = { ...currentClasses, ...stakingTabClasses };
+    }
+    if (currentTab === "2") {
+        positionTabClasses = { ...currentClasses, ...positionTabClasses };
+    }
+
+    const boxStyles = {
+        background: `linear-gradient(
+            180deg,
+            ${theme.palette.custom.elevation12} 0%,
+            ${theme.palette.custom.elevation12} 100%
+        ), ${theme.palette.background.default}`,
+        padding: 2,
+        borderRadius: 1,
+        "& .even": {
+            background: `linear-gradient(
+                180deg, ${theme.palette.custom.elevation3} 0%,
+                ${theme.palette.action.hover} 100%,
+                ${theme.palette.custom.elevation1} 100%
+            ), ${theme.palette.background.default}`,
+        },
+        "& .customRow": {
+            fontFamily: theme.typography.subtitle1.fontFamily,
+            borderRadius: 1,
+        },
+        "& .headerClass": {
+            fontFamily: "JetBrains Mono",
+        },
+        "& .MuiDataGrid-cell:focus-within": {
+            outline: "none",
+        },
+        "& .odd.MuiDataGrid-row:hover": {
+            background: `linear-gradient(
+                180deg, ${theme.palette.custom.elevation12} 0%,
+                ${theme.palette.custom.elevation12} 100%
+            ), ${theme.palette.background.default}`,
+        },
+        "& .even.MuiDataGrid-row:hover": {
+            background: `linear-gradient(
+                180deg, ${theme.palette.custom.elevation3} 0%,
+                ${theme.palette.action.hover} 100%,
+                ${theme.palette.custom.elevation1} 100%
+            ), ${theme.palette.background.default}`,
+        },
+    };
```

Comment:
Are these styles ever used elsewhere? -- If so can we abstract them out, if not and they are purely local is this the right place for them or should there be an additional file for just styles, It's a good page of content that shouldn't be altered much.